### ISSUE TITLE
Access transformers from dependencies are not auto updated

### ISSUE THREAD

**Issue by bdew** [MinecraftForge/ForgeGradle#355]
I've noticed this when trying to figure out why JEI (which is in deobfCompile of my mod) was crashing.

FG was using some very outdated version of the AT config from JEI, the file in build/depAts was "jei_at.cfg_jei_1.9-3.0.3.166_at.cfg" while the version in my dependencies was 3.2.5.179

Running setupDecompWorkspace was just reusing the old file. Only after a clean it was updated to a new version.

**Comment by darkhax** [comment]
I have noticed this issue in my workspace as well, however it's not exactly the same. I had JEI added as a dependency, and it worked as expected, however like you mentioned updating the version would not cause it to update the AT. Additionally, after adding Bookshelf which also makes use of ATs, it would not extract the AT from the mod like it did with JEI. The way to get around the issue, as you said, is to do a clean build every time.

**Comment by mezz** [comment]
Is this the same issue? https://github.com/MinecraftForge/ForgeGradle/issues/312

**Comment by tterrag1098** [comment]
This is *still* broken and still extremely annoying

**Comment by DaemonUmbra** [comment]
Closing this issue as it does not appear to be applicable to the currently supported ForgeGradle branch (``FG_3.0``).

If I am incorrect in this regard, please create a new issue prefixed with ``[FG_3.0]`` and containing:
* A summary of the issue (in case the issue has mutated slightly or the original report was poorly described)
* Steps to recreate the issue
* Console output or logs (if possible)

Note that an ongoing effort to update legacy MinecraftForge branches to utilize FG_3.0 is ongoing and accepting informed contributions. (For more information [join the Discord](https://discord.gg/UvedJ9m) and  read through the #mcpconfig channel)
